products/msexchange.md

```diff
@@ -10,35 +10,65 @@ releaseDateColumn: true
 sortReleasesBy: releaseDate
 releases:
 -   releaseCycle: "2019"
+    releaseLabel: "2019 CU12 May22SU"
     support: 2024-01-09
     eol: 2025-10-14
-    latest: "15.2.986.22 CU11"
+    latest: "15.2.1118.9"
     releaseDate: 2018-10-22
 -   releaseCycle: "2016"
+    releaseLabel: "2016 CU23 May22SU"
     support: 2020-10-13
     eol: 2025-10-14
-    latest: "15.1.2375.24 CU22"
+    latest: "15.1.2507.9"
     releaseDate: 2015-10-01
--   releaseCycle: "2013 SP1"
+-   releaseCycle: "2013"
+    releaseLabel: "2013 CU23 May22SU"
     support: 2018-04-10
     eol: 2023-04-11
-    latest: "15.0.1497.33 CU23"
+    latest: "15.0.1497.36"
     releaseDate: 2013-01-09
--   releaseCycle: "2010 SP3"
+-   releaseCycle: "2010"
+    releaseLabel: "2010 SP3 UR32"
     support: 2015-01-13
     eol: 2020-10-13
-    latest: "14.3.513.0 UR32"
+    latest: "14.3.513.0"
     releaseDate: 2009-11-09
--   releaseCycle: "2007 SP3"
+-   releaseCycle: "2007"
+    releaseLabel: "2007 SP3 UR23"
     support: 2012-04-10
     eol: 2017-04-11
-    latest: "8.3.517.0 UR23"
+    latest: "8.3.517.0"
     releaseDate: 2007-03-08
--   releaseCycle: "2003 SP2"
+-   releaseCycle: "2003"
+    releaseLabel: "2003 SP2"
     support: 2009-04-14
     eol: 2014-04-08
     latest: "6.5.7654.4"
     releaseDate: 2003-09-28
+-   releaseCycle: "2000"
+    releaseLabel: "2000 SP3"
+    support: 2005-12-31
+    eol: 2011-01-11
+    latest: "6.0.6620.7"
+    releaseDate: 2000-11-29
+-   releaseCycle: "5.5"
+    releaseLabel: "5.5 SP4"
+    support: 2003-12-31
+    eol: 2006-01-10
+    latest: "5.5.2653"
+    releaseDate: 1998-02-03
+-   releaseCycle: "5.0"
+    releaseLabel: "5.0 SP2"
+    support: 2003-12-31
+    eol: 2006-01-10
+    latest: "5.0.1460"
+    releaseDate: 1997-05-23
+-   releaseCycle: "4.0"
+    releaseLabel: "4.0 SP5"
+    support: no
+    eol: yes
```

Comment:
Avoiding yes/no in YAML is good practice. https://www.bram.us/2022/01/11/yaml-the-norway-problem/

```suggestion
    support: false
    eol: true
```

Reply:
Okay, thanks for pointing out, I didn't know